airflow/cli/commands/connection_command.py

```diff
@@ -197,9 +197,9 @@ def connections_export(args):
         f.write(msg)
 
     if file_is_stdout:
-        print("\nConnections successfully exported.", file=sys.stderr)
+        print(f"\n{len(connections)} connections successfully exported.", file=sys.stderr)
```

Comment:
@Jayden-Chiu Not a blocker, How can we ensure that the output of `export connection` `export variable` and `users export` doesn't mismatch by any further changes in the codebase?

Can we add a single function to prepare a similar output?

Reply:
@utkarsharma2 good point, could you open a new PR to do that?

Reply:
@hussein-awala Sure, I'll raise that.